data/estimate.py

```diff
@@ -0,0 +1,11 @@
+def mysql_estimate_row_count(model_cls, db):
+    """ Uses the information_schema to retrieve the row count for a table. """
+    query = "SELECT table_name, table_rows FROM information_schema.tables WHERE table_schema = DATABASE() AND table_name = %s"
```

Comment:
you don't need to select out the table name here if you're using an `=` where clause

Reply:
Good point.